docs-en/chapter_introduction/algorithms_are_everywhere.md

```diff
@@ -4,10 +4,10 @@ When we hear the word "algorithm", we naturally think of mathematics. However, m
 
 Before we formally discuss algorithms, an interesting fact is worth sharing: **you have already learned many algorithms unconsciously and have become accustomed to applying them in your daily life**. Below, I will give a few specific examples to prove this point.
 
-**Example 1: Looking Up a Dictionary**. In a standard dictionary, each word corresponds to a phonetic transcription and the dictionary is organized alphabetically based on these transcriptions. Let's say we're looking for a word that begins with the letter $r$. This is typically done in the following way:
+**Example 1: Looking Up a Dictionary**. In an English dictionary, words are listed alphabetically. Suppose we're searching for a word that starts with the letter $r$. This is typically done in the following way:
 
 1. Open the dictionary around its midpoint and note the first letter on that page, assuming it to be $m$.
-2. Given the sequence of words following the initial letter $m$, estimate where words starting with the letter $r$ might be located within the alphabetical order.
+2. Since $r$ comes after $m$ in the pinyin alphabet, we exclude the first half of the dictionary and narrow the search to the second half.
```

Comment:
pinyin alphabet -> alphabet

Reply:
err... my bad, and thank you so much for your comment Krahets!